irods/message/__init__.py

```diff
@@ -163,11 +173,25 @@ def _recv_message_in_len(sock, size):
             if getattr(e, 'winerror', 0) != 10045:
                 raise
             buf = sock.recv(size_left)
+
+        # Break out of the loop
```

Comment:
This prevents an infinite loop. If the call to recv() returns an empty buffer break out of the loop.

Reply:
Very good. Could you put a statement like this in the code comment for future readers? IMO, your Github comment here, verbatim, would be fine, actually.

Reply:
Looks like @trel beat me to it :) Thanks!